src/Components/Registrationform1/Form.jsx

```diff
@@ -0,0 +1,64 @@
+import style from "./Form.module.scss";
+
+const Form1 = ({ fields, handleMainFormChange }) => {
+  return (
+    <div className={style.formelementcontainer}>
+      <div className={style.inputcontainer}>
+        <input
+          type="text"
+          className={style.input}
+          value={fields.teamName}
+          name="teamName"
+          onChange={(event) => handleMainFormChange(event)}
+        />
+      </div>
+      <div className={style.inputcontainer}>
+        <input
+          type="text"
+          className={style.input}
+          value={fields.name}
+          name="name"
+          onChange={(event) => handleMainFormChange(event)}
+        />
+      </div>
+      <div className={style.inputcontainer}>
+        <input
+          type="text"
+          className={style.input}
+          value={fields.phone}
+          name="phone"
+          onChange={(event) => handleMainFormChange(event)}
+        />
+      </div>
+      <div className={style.inputcontainer}>
+        <input
+          type="text"
+          className={style.input}
+          value={fields.email}
+          name="email"
+          onChange={(event) => handleMainFormChange(event)}
+        />
+      </div>
+      <div className={style.inputcontainer}>
+        <select
+          className={style.select}
+          name="gender"
+          onChange={(event) => handleMainFormChange(event)}
+        >
+          <option defaultValue={fields.gender} className={style.option}>
+            Gender
+          </option>
+          <option className={style.option} value="Male">
+            Male
+          </option>
+          <option className={style.option} value="Female">
+            Female
+          </option>
+        </select>
```

Comment:
1) Instead of putting value as Name, Email etc put that as empty and write Name, Email etc in placeholder instead because when someone clicks on the input box, the user has to clear the value present earlier i.e. Name, Email etc whatever. and then can write in the input box. 

2) Same for gender, disable the option `Gender` as user can now select `Gender` as one of the option, which shouldn't be allowed. Also keep `selected` property on the `Gender` option as it should be displayed in the select tag.

3) Place the value property on select tag not on the option tag to find which tag is selected